packages/expo-constants/ios/EXConstants/EXConstantsService.m

```diff
@@ -418,5 +419,23 @@ + (NSString *)installationId
   return uuid;
 }
 
++ (NSDictionary *)appConfig
+{
+  NSString *path = [[NSBundle mainBundle] pathForResource:@"app" ofType:@"config"];
+  if (path) {
```

Comment:
maybe if there is no path we log a warning similar to discussed above for FileNotFoundException